README.md

```diff
@@ -1,4 +1,4 @@
-# Cobalt [![Build Status](https://img.shields.io/badge/-Build%20Status-blueviolet)](https://github.com/youtube/cobalt_sandbox/blob/main/BUILD_STATUS.md)
+# Cobalt [![Build Status](https://img.shields.io/badge/-Build%20Status-blueviolet)](https://github.com/youtube/cobalt_sandbox/blob/main/BUILD_STATUS.md) [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/youtube/cobalt_sandbox/badge)](https://api.securityscorecards.dev/projects/github.com/youtube/cobalt_sandbox)
```

Comment:
The link here takes you directly to a text dump. Is there something better that we can link to?

Reply:
I did what was recommended in https://github.com/ossf/scorecard-action#scorecard-badge, it appears to be done by design.  Where do you want the link take you, is it https://github.com/youtube/cobalt_sandbox/security/code-scanning ?

Reply:
You can see the link in the markdown, it's [https://api.securityscorecards.dev/projects/github.com/youtube/cobalt_sandbox](https://api.securityscorecards.dev/projects/github.com/youtube/cobalt_sandbox/badge)%5D(https://api.securityscorecards.dev/projects/github.com/youtube/cobalt_sandbox)

Reply:
That is correct, this is per https://github.com/ossf/scorecard-action#scorecard-badge

Reply:
Sorry, I misread your previous question. I think https://github.com/youtube/cobalt_sandbox/security/code-scanning is a great place to have the link take you

Reply:
done